cg/meta/demultiplex/utils.py

```diff
@@ -38,12 +38,23 @@ def get_lane_from_sample_fastq(sample_fastq_path: Path) -> int:
 def get_sample_fastqs_from_flow_cell(
```

Comment:
This function should probably be split into several smaller ones. Like `fetch_fastq_files_for_bcl_convert_flow_cell` and yada yada. Something to do later